frontend/amundsen_application/static/js/components/Lineage/Graph/constants.ts

```diff
@@ -6,7 +6,7 @@ export const LINEAGE_SCENE_MARGIN = {
 };
 export const CHART_DEFAULT_DIMENSIONS = { width: 1280, height: 1024 };
 export const CHART_DEFAULT_LABELS = { upstream: '', downstream: '' };
-export const UPSTREAM_LABEL_OFFSET = 150;
+export const UPSTREAM_LABEL_OFFSET = 200;
```

Comment:
what exactly is this offset used for?

Reply:
make that change because the front test is failing precisely because of that property, I thought it should be expanded since now the title is larger, but the front test still failing.

Reply:
Thanks for the explanation. A colleague at work actually send this OR to me and I thought it was a change his team made to our fork of Amundsen and thought it needed a review from me, so I accidentally began my review. Apologies for the confusion.

But this feature is great! I am looking forward to it!